R/resample.R

```diff
@@ -13,6 +13,9 @@
 #' @param control A [control_resamples()] object used to fine tune the resampling
 #'   process.
 #'
+#' @param eval_times A numeric vector of time points where dynamic event time
```

Comment:
this could be inherited from `tune_grid()`

Reply:
I noticed that and plan to do a small PR that maximizes inheritance across the `tune_*()` functions and `fit_resamples()`.

Reply:
See #622